Library/Formula/tomcat-runner.rb

```diff
@@ -0,0 +1,18 @@
+class TomcatRunner < Formula
+  desc "Run Tomcat without an installed distribution"
+  homepage "https://github.com/jsimone/webapp-runner"
+  url "https://repo1.maven.org/maven2/com/github/jsimone/webapp-runner/8.0.24.0/webapp-runner-8.0.24.0.jar"
+  sha256 "906ca6902dd481ad1b7e0c5287daefc81e111b70b6f4ec37c1bf03b48e56099d"
+
+  def install
+    libexec.install Dir["*"]
+
+    bin.write_jar_script libexec/"webapp-runner-#{version}.jar", "tomcat-runner"
```

Comment:
@mikemcquaid, thanks for looking into this!

Yeah, I would name it `webapp-runner` at the first sight, but clearly this tool only support tomcat, which is explicit in its README:

> Webapp runner is designed to allow you to launch an exploded or compressed war that is on your filesystem into a tomcat container with a simple java -jar command. It supports the following version of Tomcat:

> Tomcat 7: tomcat7 branch
> Tomcat 8: master branch

Also, isn't `webapp-runner` too broad? considering there's also jetty-runner which support jetty, and a lot more different webapps/platforms beyond the Java ecosystem.